cmd/root.go

```diff
@@ -93,13 +98,21 @@ func initConfig() {
 	viper.MergeInConfig()
 
 	if isConfigExtends() {
-		filename := filepath.Base(getExtendsPath())
-		extension := filepath.Ext(getExtendsPath())
-		name := filename[0 : len(filename)-len(extension)]
-		viper.SetConfigName(name)
-		viper.AddConfigPath(filepath.Dir(getExtendsPath()))
-		err := viper.MergeInConfig()
-		check(err)
+		if isValidUrl(getExtendsPath()) {
+			uri := getExtendsPath()
+			baseRemoteConfig := fetchRemoteExtends(uri)
+			viper.SetConfigType("yaml")
+			err := viper.MergeConfig(baseRemoteConfig)
```

Comment:
Just merge is not enough here. We need to install that config. Thus way we'll have local changes in `.git/hooks`

Reply:
How local config will interact with remote?

Reply:
> Just merge is not enough here. We need to install that config. Thus way we'll have local changes in .git/hooks

I don't exactly know the answer to this. I assumed that given it's pulled in an merged when the command is ran that whenever the regular config is installed this would be too. I'll have to take some more time to really understand how the config is installed to definitively answer that question.